docs/pressrelease-wot-rec.html

```diff
@@ -52,11 +52,11 @@ <h2>Web technologies enable the full potential of the Internet of Things by inte
     integration across Internet of Things platforms and
     applications. 
     </p>
-    <p>Interoperability of devices and services for the Internet of Things (IoT) with cloud platforms and services 
-    at a worldwide scale is a huge challenge.<br> 
-    
+    <p><br> 
+   The diversity of technologies in Internet of Things (IoT), such as protocols and data models, typically causes high integration and maintenance 
+	    costs in IoT projects.
   The mission of the <a href="https://www.w3.org/WoT/">Web of Things</a> activities 
-  is to counter the fragmentation of the Internet of Things 
+  is to counter these fragmentation of the IoT
   by forming a Web-based abstraction layer capable of interconnecting existing IoT platforms, devices, 
   and cloud services and complementing available standards. It reduces the
     risk of both investors and customers and enables and encourages
```

Comment:
This sentence included 4 time "and": It reduces the risk of both investors **and** customers **and** enables **and** encourages exponential growth of open markets of devices **and** services.

Reply:
>  is to counter these fragmentation of the IoT
Suggest not abbreviate IoT in the first introduction part of the press release.
"these" would be plural, do we want to call out multiple fragmentations?

Reply:
I suggest, this kind of refinement can be decided by the W3C MarkCom team.

Reply:
I agree that we have many "and"s and the sentences don't flow.

What about:
The  <a href="https://www.w3.org/WoT/">Web of Things</a> counters the fragmentation of the Internet of Things by defining a Web-based abstraction layer for existing platforms, devices, gateways and services.
By complementing existing standards it reduces the risk for investors and customers and enables  rapid growth of open markets for devices and services.

Reply:
much better, we have one "and" less ;-) How about this:

The W3C Web of Things counters the fragmentation of the Internet of Things by defining a Web-based abstraction layer for, e.g., existing platforms, devices, gateways and services. By complementing existing standards, it reduces risk for investors and customers as well as enabling rapid growth of open markets for devices and services.